### PR TITLE
network: allow single-node privnet setup

### LINKED ISSUE #595: Allow single-node privnet setup

### PR DESCRIPTION
**Pull request by fyrchik**
Closes #595.

### CODE REVIEW

**Comment by roman-khimov** [comment]
Please make sure we're compatible with neo-local wrt this setup.

**Comment by fyrchik** [comment]
I have dumped 1600 blocks from existing `neo-local` setup and successfully imported.

File: Makefile [hunk 1]
```diff
@@ -107,6 +107,10 @@ env_up:
 	@echo "=> Bootup environment"
 	@docker-compose -f $(DC_FILE) up -d node_one node_two node_three node_four
 
+env_single:
```
Comment (roman-khimov) [inline Makefile]: Will it go down on `make env_down`?

Comment (fyrchik) [reply Makefile]: Yes.

**Pull request merged**